vips/vip-427/bscmainnet.ts

```diff
@@ -0,0 +1,164 @@
+
+export const tokens = {
+  USDM: {
+    address: "0x7715c206A14Ac93Cb1A6c0316A6E5f8aD7c9Dc31",
+    decimals: 18,
+    symbol: "USDM",
+  },
+  wUSDM: {
+    address: "0xA900cbE7739c96D2B153a273953620A701d5442b",
+    decimals: 18,
+    symbol: "wUSDM",
+  },
+};
+
+export const newMarket = {
+  vToken: {
+    address: "0x183dE3C349fCf546aAe925E1c7F364EA6FB4033c",
+    name: "Venus wUSDM (Core)",
+    symbol: "vwUSDM_Core",
+    underlying: tokens["wUSDM"],
+    decimals: 8,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: COMPTROLLER,
+  },
+  riskParameters: {
+    collateralFactor: parseUnits("0.75", 18),
+    liquidationThreshold: parseUnits("0.78", 18),
+    supplyCap: parseUnits("5000000", 18),
+    borrowCap: parseUnits("4000000", 18),
+    reserveFactor: parseUnits("0.25", 18),
+    protocolSeizeShare: parseUnits("0.05", 18),
+  },
+  initialSupply: {
+    amount: parseUnits("10000", 18),
+    vTokenReceiver: "0xFfCf33Ed3fc6B7eC7d4F6166cC1B86d4F42Af192",
+  },
+  interestRateModel: {
+    address: "0x40a0DEC6AcA207F45212B14dE1312cEae6FB3E5A",
+    base: "0",
+    multiplier: "0.06875",
+    jump: "2.5",
+    kink: "0.8",
+  },
+};
+
+const vip427 = (overrides: { chainlinkStalePeriod?: number }) => {
+  const meta = {
+    version: "v2",
+    title: "[zkSync] Add wUSDM market",
+    description: ``,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+
+  const chainlinkStalePeriod = overrides?.chainlinkStalePeriod || CHAINLINK_STALE_PERIOD;
+
+  return makeProposal(
+    [
+      // Oracle config
+      {
+        target: CHAINLINK_ORACLE,
+        signature: "setTokenConfig((address,address,uint256))",
+        params: [[tokens["USDM"].address, USDM_CHAINLINK_FEED, chainlinkStalePeriod]],
+        dstChainId: LzChainId.zksyncmainnet,
+      },
+      {
+        target: RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            tokens["USDM"].address,
+            [CHAINLINK_ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero],
+            [true, false, false],
+          ],
+        ],
+        dstChainId: LzChainId.zksyncmainnet,
+      },
+      {
+        target: RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            tokens["wUSDM"].address,
+            [WUSDM_ERC4626_ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero],
+            [true, false, false],
+          ],
+        ],
+        dstChainId: LzChainId.zksyncmainnet,
+      },
+
+      // Market
+      {
+        target: newMarket.vToken.address,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["28800"],
```

Comment:
This is defined in seconds, and we usually set 24 hours. So, it would be 86400 seconds

```suggestion
        params: ["86400"],
```

Reply:
https://github.com/VenusProtocol/vips/pull/462/commits/d712fca4bf3ad0f19a32d0dd6eaaf9d5b83757bb